### DISCUSSION TITLE
6000th commit 🎉🎉🎉🥳

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#6359] [General]
spotted on Mastodon:

![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/6ab9a9da-61b3-4fa9-a9bb-21dac820ad7e)

**Comment by math-GH** [comment]
6666 Commits ✅

<img width="965" height="234" alt="grafik" src="https://github.com/user-attachments/assets/84b28363-4c2a-4cd0-8a8c-981552487c0d" />

<img width="1370" height="775" alt="grafik" src="https://github.com/user-attachments/assets/2b612219-aad5-4a65-91b5-9ea22e05d676" />

**Comment by lukasMega** [comment]
<img width="1417" height="550" alt="scr 2025-09-01 at 14 41 40" src="https://github.com/user-attachments/assets/d1b07fa4-8b32-4b71-8810-0c01cc0da557" />
congrats 🎉 

// update: I noticed a bug ?! it says `a year ago` :D

**Reply by Inverle** [reply]
A bug in github's feed maybe (the date is of the post not the comment)

**Reply by math-GH** [reply]
Github uses the first post content and date everytime a comment is made. It is a bit strange.
